packages/libs/error-stack/src/frame.rs

```diff
@@ -200,9 +200,13 @@ impl Provider for Frame {
 
 impl fmt::Debug for Frame {
     fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
```

Comment:
Can you also attach an example of what this alternate formatting looks like when debugging a Report?

Reply:
Updated the description to show this output as well